static/img/logo.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you please rename to something more descriptive, ideally it would be the same name as in branding package. https://graphql.org/brand/

Also please rename `static/img/docusaurus.png` since it's GraphQL logo now.
BTW. do we need both png and SVG, can we just use SVG everywhere?

Reply:
Can we rename it to `graphql-logo.svg` because right now it's not clear what's inside.
Site's logo, graphql-js's logo or graphql's logo.

Reply:
> graphql-js's logo or graphql's logo

is there a different logo for `graphql-js`?